### ISSUE TITLE
Debian 

### ISSUE THREAD

**Issue by AutomationD** [utmapp/UTM#6747]
**Configuration (required)**
* UTM Version: 4.5.4.101
* OS Version: 18.0.1
* Device Model: iPad Pro 4th Gen
* Is it jailbroken (name jailbreak used)?  No.
* How did you install UTM? App Store (UTM SE)

When UTM loses focus port forwarding stops working.
I'm forwarding ssh port to connect from a ssh client, running it in stage manager and it works fine. But as soon as I stop using it it the forwarding stops working. It's important to note that I when I get back to the VM I still see it active (not paused/suspended or stopped).

---
Please let me know if I can provide any addition debug info, happy to do so.

**Comment by AutomationD** [comment]
Hi @osy,

The question is not the pausing itself (background GPS or silent Audio works fine for me), but the resuming part. Any idea why it doesn't restart forwarding when I'm back to the app and it is required to restart the app?

**Comment by osy** [comment]
Yeah it's weird, I don't know what iOS is doing here. We don't do anything special to enable port forwarding. My guess is that something weird is happening at the socket level when iOS automatically suspends the process. Maybe the internal sockets timed out and that's not getting propagated to the guest.

**Comment by AutomationD** [comment]
I'm wondering if there is a way to work around that, such as catch the restore of the process and re-establish the port forwarding if it's not active? But I have zero competence here, so those are just my hopes and wishes 🥲

**Comment by osy** [comment]
The problem is that iOS provides very little visibility to what it is doing with process suspension. This could take quite a while to debug.

**Comment by AutomationD** [comment]
Makes sense. 

Again, I'm a compete noob in iOS so feel free to ignore.

A quick google revealed that VoIP sockets are treated differently and not shut down. I assume it hasn't been used because Apple won't allow this?

Also, could we explore something like applicationDidEnterBackground to save the state of the socket?

If there is any testing I can do to help this move forward please let me know. I also have Apple Dev certificate and can build the app locally if needed.

**Comment by introspection3** [comment]
@AutomationD @osy dear all,you can use vpn in utm,it can't hold sokcet
